src/metadata/v57.json

```diff
@@ -2027,5 +2027,19 @@
     "xmlTag": "workflowTasks",
     "key": "name",
     "excluded": true
+  },
+  {
+    "directoryName": "uiObjectRelationConfigs",
+    "inFolder": false,
+    "metaFile": false,
+    "suffix": "uiObjectRelationConfig",
+    "xmlName": "UIObjectRelationConfig"
+  },
+  {
+    "directoryName": "timelineObjectDefinitions",
+    "inFolder": false,
+    "metaFile": false,
+    "suffix": "timelineObjectDefinition",
+    "xmlName": "TimelineObjectDefinition"
```

Comment:
**change:** I would prefer this declaration to be on top of the file please

End of the file is related to inFile special metadata

Reply:
Sure, working on it, thanks for your review.